docs/src/recipes/common-patterns.md

```diff
@@ -1,36 +1,72 @@
 # Common patterns
 
+As a specification, biscuit does not mandate specific ways to use datalog. As far as authorization logic is concerned,
+there are no built-in facts with specific  behaviour. That being said, some patterns are common and while not part of
+the spec, are codified in libraries and tools. Finally, using specific fact names can help with reducing token size.
+
 ## Expiration check
 
-If the authorizer provides the current time when executing policies,
-the token can then carry an expiration date:
+The CLI and the rust library (among others) use the `time()` fact to represent the instant in time where the token is used.
+This provides a way to encode expiration dates in tokens:
+
+<bc-datalog-editor readonly="readonly">
+   <code>
+check if time($time), $time <= 2022-03-30T20:00:00Z;
+  </code>
+</bc-datalog-editor>
+
+It only depends on the authorizer and tokens using the same fact name (here, `time()`). It would work with other fact names,
```

Comment:
```suggestion
Expiration checks require the authorizer and tokens to use the same fact name (here, `time()`). It would work with other fact names,
```
For clarity purposes